Dockerfile

```diff
@@ -1,7 +1,14 @@
 FROM python:3.9.16 AS py-deps
-COPY ./Pipfile ./Pipfile.lock ./
-RUN pip install -U pip setuptools pipenv && \
-    pipenv install --deploy --system --ignore-pipfile --clear
+
+RUN pip install -U pip setuptools wheel
+RUN pip install pdm
+
+
+WORKDIR /python
+COPY pyproject.toml pdm.lock README.md ./
+
+# Some little PEP582 magic
```

Comment:
I recognize this from somewhere… 🤔

Reply:
Everybody comments this I don't know what you're saying